### PR TITLE
Merge the correct lyrics for the poem into main

### PR DESCRIPTION
**Pull request by shaktipn**
trying to resolve conflict
update: resolved

### CODE REVIEW

File: exercises/resolving-merge-conflict/MaryHadALittleLamb.md [hunk 1]
```diff
@@ -2,31 +2,34 @@
...
 
-When the blazing sun is gone,
-When he nothing shines upon,
-Then you show your little light,
-Twinkle, twinkle, all the night.
+It made the children laugh and play,
+Laugh and play, laugh and play.
+It made the children laugh and play
+To see a lamb at school.
+
```
Comment (sudarshan-dodiya) [inline exercises/resolving-merge-conflict/MaryHadALittleLamb.md:23]: Why extra line here?

**Review by sudarshan-dodiya — CHANGES_REQUESTED** [review]
- Please remember to add me for a review on PR as well.
- The title does not state what is actually happening in the PR, please update the same. 
- Make sure if feedback is given at one place, it is addressed everywhere it is applicable.

File: exercises/resolving-merge-conflict/MaryHadALittleLamb.md (hunk 1 above)
Comment (shaktipn) [reply exercises/resolving-merge-conflict/MaryHadALittleLamb.md:23]: Got it. I didn't check the feedbacks yet actually. But I will from now

Comment (shaktipn) [reply exercises/resolving-merge-conflict/MaryHadALittleLamb.md:22]: Removed extra lines at end. The blank lines in middle are separation for paragraphs of poem.

**Review by sudarshan-dodiya — APPROVED** [review]
Here, the title is over descriptive. 
- The PR is not merging anything. PR title should contain small description on what we are doing in this PR.
- You should never add a branch where the PR is going to get merged in commit message or title, what if I change the name from `main` to `master`?

**Pull request merged**